### PR TITLE
small tweaks to settings.js which is used from main.html

### PR DESCRIPTION
**Pull request by janedegtiareva**
to make local ui development experience better

### CODE REVIEW

**Comment by vgrichina** [comment]
I think it makes sense to align this with changes I've been making to make GH pages work well for:
https://github.com/nearprotocol/near-chess
https://github.com/nearprotocol/near-shell/commits/support-gh-pages

I think we need to agree on how to solve https://github.com/nearprotocol/near-shell/issues/21, cause otherwise it's very inconvenient to switch between local / shared devnet.

**Comment by janedegtiareva** [comment]
> I think it makes sense to align this with changes I've been making to make GH pages work well for:
> https://github.com/nearprotocol/near-chess
> https://github.com/nearprotocol/near-shell/commits/support-gh-pages
> 
> I think we need to agree on how to solve #21, cause otherwise it's very inconvenient to switch between local / shared devnet.

re GH pages - settings.js looks pretty much identical except for setting the fiddle cookie. Can we actually change that? I don't want the real app code to say "fiddle" anywhere. Why can't we just read the values from the settings.js directly and bypass fiddle config? 

#21 - switching between envs is just passing node url at this point. I do not think that fixing this is higher priority than key management, so will not work on it right now.

**Comment by vgrichina** [comment]
> Why can't we just read the values from the settings.js directly and bypass fiddle config?

I agree we shouldn't use cookie, that's just hack to make stuff work quickly. 

I'm thinking that it makes sense to change it as part of doing https://github.com/nearprotocol/near-shell/issues/21, so that e.g. we have few .json config files (one per environment).

**Comment by janedegtiareva** [comment]
> > Why can't we just read the values from the settings.js directly and bypass fiddle config?
> 
> I agree we shouldn't use cookie, that's just hack to make stuff work quickly.
> 
> I'm thinking that it makes sense to change it as part of doing #21, so that e.g. we have few .json config files (one per environment).

Cool! This MR is going in that direction by adding 2 settings files. You still need to specify which one to use manually, but this is something that can be iterated on.

File: blank_project/src/index.html [hunk 1]
```diff
@@ -7,11 +7,11 @@
 </head>
 <body style="background: #fff">
   <div class="container">
-    Contract says:
+    Contract says 2:
```
Comment (vgrichina) [inline blank_project/src/index.html]: @janedegtiareva hmm, seems like typo

Comment (janedegtiareva) [reply blank_project/src/index.html]: good catch. I use these sometimes to make sure i am looking at the latest stuff and forget to undo lol

**Pull request merged**